sapp/ui/schema.py

```diff
@@ -213,6 +213,14 @@ def resolve_issues(
             .where_issue_instance_id_is(issue_instance_id)
             .get()
         )
+
+        other_issues = Instance(session, DBID(run_id)).get()
+        for issue in issues:
+            for other_issue in other_issues:
+                if issue == other_issue:
+                    continue
+                issue.compute_and_update_similarity_with(other_issue)
```

Comment:
So this algorithm is exponential in big O notation, which is not going to work for any reasonably sized database. We're going to have to come up with a better method than checking every issue against every other issue. A few potential ideas:

1. If we could come up with some sort of hash that can be used as a similarity measure, this could be reduced to an O(log(n)) lookup in a sorted list
2. If we switched to only showing similar issues on the issue screen, this could be reduced to O(n) per query. This still doesn't make me feel great, but for pretty much every usecase of SAPP it'll probably be OK

Reply:
What are your opinions on computing this as a pipeline step and storing it in the db? If we can't do that, I will resort to 2 as I can't come of a hash!

Reply:
@gbleaney @0xedward I came up with a hash, https://www.learndatasci.com/tutorials/building-recommendation-engine-locality-sensitive-hashing-lsh-python/

The query time would be O(1) for each issue (which is something fantastic), but it does have a build up time complexity of O(n). I honestly wanna do this O(n) computation as a pipeline step, store it somewhere, and then just query!

Also for this we may have to add an additional external dependency. But, even if we can't, I can implement this natively.